static/js/components/tiles/disaster_event_map_tile.tsx

```diff
@@ -341,10 +351,14 @@ export function DisasterEventMapTile(
       (geoDcid: GeoJsonFeatureProperties) =>
         redirectAction(geoDcid.geoDcid) /* redirectAction */,
       (place: NamedPlace) => place.name || place.dcid /* getTooltipHtml */,
-      () => true /* canClickRegion: allow all regions to be clickable */,
+      () =>
+        !isPlaceBaseMap /* canClickRegion: allow all regions to be clickable if not using place base map */,
```

Comment:
as well as if it's the same place (on leaf maps, we currently allow clicks to self which resets the view, etc. we should disallow clicks).